packages/commonwealth/server/scripts/backfillNamespaceAddresses.ts

```diff
@@ -0,0 +1,86 @@
+import { PinoLogger } from '@hicommonwealth/adapters';
```

Comment:
Just a thought... How many namespaces are there? IF the list is small, a temporary seed table could have allowed a migration script, which is often preferred over manual scripts.

Reply:
To be extra safe I just converted it all to JS and put all the backfill logic in a migration. Took longer but now we don't have to worry about running the script or hardcoding anything.